src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,34 @@
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $block-size * 7 + $calendar-gap * 6;
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $block-size;
+    height: $block-size;
+    background-color: #eee;
+    border: 1px #000 solid;
+    transition: transform 0.5s, background-color 0.5s ;
+
+    &:hover {
+      background-color: #febfcb;
+      transform: translateY(-20px);
+      cursor: pointer;
+    }
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index}):before {
+        content: "#{$index}";
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        color: #000;
```

Comment:
these styles you can apply to the calendat__block, you don't need to repeat them in the loop.  Read the checklist one more time.